src/screens/Customers/components/TariffPlanInfo.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { createStyles, withStyles, Theme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { CardPropsInfo } from '../../../components/OutlinedCardList';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      marginTop: '10px',
+      border: '1px solid',
+      borderColor: theme.palette.divider,
+      borderRadius: '6px',
+    },
+    content: {
+      flex: 1,
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      padding: '10px',
+    },
+    item: {
+      width: '100%',
+      color: theme.palette.text.primary,
+    },
+    label: {
+      whiteSpace: 'pre-line',
+      lineHeight: '1.2',
+      marginTop: '5px',
+    },
+  });
+
+interface TariffPlanInfoProps {
+  classes: { root: string; content: string; item: string; label: string; };
+  undefinedAmount: string;
+  meterType: string;
+  fixedTariff: string;
+  unitTariff: string;
+  freeUnits: string;
+}
+
+function TariffPlanInfo(props: TariffPlanInfoProps) {
+  const { classes, undefinedAmount, meterType, fixedTariff, unitTariff, freeUnits } = props;
+
+  let tariffInfo : CardPropsInfo[];
+  //TODO: swap with something like customer.inactive
+  if (meterType === "Inactive") {
+    tariffInfo = [
+      { number: undefinedAmount, label: 'Fixed\nTariff', unit: '' },
+      { number: undefinedAmount, label: 'Unit\nTariff', unit: '' },
+      { number: undefinedAmount, label: 'Free\nUnits', unit: '' },
+    ];
+  } else {
+    tariffInfo = [
+      { number: fixedTariff , label: 'Fixed\nTariff', unit: 'Ks' },
+      { number: unitTariff, label: 'Unit\nTariff', unit: 'Ks' },
+      { number: freeUnits, label: 'Free\nUnits', unit: 'kWh' },
+    ];
+  }
+
+  return (
```

Comment:
I'm not sure why you re-wrote the tariff info when it was originally just one line:
`<OutlinedCardList info={tariffInfo} primary={false} columns />` 
and it appears that the only difference is that the text is now on the bottom (which could easily be implemented in `OutlinedCardList` by passing in a prop). I understand why it's clean to have this as a dedicated component for the `tariffInfo` logic, but I think it'd make sense if you replace this with the original one-liner.